### DISCUSSION TITLE
Need Help Uploading Student Certificate: The image you uploaded appears to be from a third party.

### DISCUSSION THREAD

**Discussion by Hamzaless** [community/community#62669] [GitHub Education] [answered]
### Hi everyone,

I am a student from Turkey. I do not have a school email, but I possess a student certificate issued by the Ministry of Education. When I convert it to JPG format and upload it to the website, I encounter the following error message: "The image you uploaded appears to be from a third party. Please submit academic affiliation documentation obtained directly by your school." Additionally, due to the absence of a school email, I receive the following error: "If your school does not provide an academic email, then you must enable the use of your device camera to capture your image."

I kindly request your assistance in resolving this matter. Thank you for your help. 💖

**Comment by YnlzAtM** [comment]
çözüm buldun mu hiç

**Reply by Hamzaless** [reply, marked as answer]
obsten sanal kamera açıp fotoğrafı ekrana vererek geçmiştim sanırım

**Comment by Arif-Ata-Kosker** [comment]
Kanka pdf yüklesene ayrıca nasıl yüklicez ben yükleyemedim bile chrome ve microsoft edge de bu özellik yok

**Reply by iphysicc** [reply]
Nasıl çözdünüz?

**Comment by YnlzAtM** [comment]
Sen kimdin acaba kusura bajma simdiden

2 Nis 2025 Çar 23:45 tarihinde Arif Ata Köşker ***@***.***>
şunu yazdı:

**Reply by Arif-Ata-Kosker** [reply]
Benimde böyle bir sorunum vardı ama çözüldü başvuru da eksiklerim varmış

**Comment by YnlzAtM** [comment]
Githubdan geldiğini sonradan fark ettim kusura bakma kral. Ben hala çözemedim valla
________________________________
